### DISCUSSION TITLE
Matery主题使用waline出现字体大小问题

### DISCUSSION THREAD

**Discussion by Steven-WZH** [walinejs/waline#1596] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/113268133/202876641-4ad3e017-d02d-4e44-bb04-15cbb92ff4ae.png)

昵称 邮箱 和 网址 的字体都变小了

**Comment by Mister-Hope** [comment, marked as answer]
That maybe your own problem. The theme may override default style for label, you may check it.

For size consideration, we are not expecting to make a full style normalize for just a "comment component". i.e.: if your theme modify default brower style, you are expected to revert the changes for Waline by adding extra styles.

**Reply by Steven-WZH** [reply]
> That maybe your own problem. The theme may override default style for label, you may check it.
> 
> For size consideration, we are not expecting to make a full style normalize for just a "comment component". i.e.: if your theme modify default brower style, you are expected to revert the changes for Waline by adding extra styles.

thank u very much , i'll check the theme
